admin/lint_json.m

```diff
@@ -28,7 +38,13 @@ function lint_json(filesin, outputfile)
         % Filter doc files
         flist = filter_list(flist, @(x)(startsWith(x.name,'doc_')));
         flist = arrayfun(@(file)(fullfile(file.folder, file.name)), flist, 'UniformOutput', false);
-        files = [files; flist];
+        files = unique([files; flist]);
+    end
+
+    for i = 1:numel(exclude)
+        excl = dir(exclude{i});
+        excl = arrayfun(@(file)(fullfile(file.folder, file.name)), excl, 'UniformOutput', false);
```

Comment:
is it possible to restructure the `filelist - exludefiles` operation so you only need one call to the arrayfun on the filtered list to build build the full paths of the retained files? 

e.g. build an initial list of unique `file`s (retaining filename and folder information), using setdiff to remove each of the excluded folders/files, and then building an output list of the full paths?